tests/test_adjointA.py

```diff
@@ -58,7 +59,8 @@ def test_acoustic(mkey, dimensions, time_order, space_order, nbpml, fix_dim):
                                 space_order=space_order)
 
     # Set fixed ("baked-in") time dimension if requested
-    time.size = solver.source.nt if fix_dim else None
+    if fix_dim:
+        devito.time = FixedDimension('time', size=solver.source.nt)
```

Comment:
Hmmm, not sure I completely like this way of overriding the global time dimension, but at least it is (painfully) explicit what is happening. I'd be interested in what "dynamic dispatch voodoo" was envisioned here, but it's ok for now IMHO.

Reply:
Maybe an issue?